### ISSUE TITLE
Broken Travis build - can't find js-beautify.js

### ISSUE THREAD

**Issue by bpaduch** [ampproject/docs#695]
See https://travis-ci.org/ampproject/docs/builds/276831477

```text
npm ERR! Linux 4.11.6-041106-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v7.10.1/bin/node" "/home/travis/.nvm/versions/node/v7.10.1/bin/npm" "install"
npm ERR! node v7.10.1
npm ERR! npm  v4.2.0
npm ERR! path /home/travis/build/ampproject/docs/node_modules/js-beautify/js/bin/js-beautify.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/travis/build/ampproject/docs/node_modules/js-beautify/js/bin/js-beautify.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/travis/build/ampproject/docs/node_modules/js-beautify/js/bin/js-beautify.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/.npm/_logs/2017-09-18T12_43_09_727Z-debug.log
```

Issue is related to the `js-beautify.js` node module per https://github.com/beautify-web/js-beautify/issues/1247.

**Comment by bpaduch** [comment]
This issue is fixed; however new breakage:

```text
0.31s$ grow build
Traceback (most recent call last):
  File "/home/travis/.local/bin/grow", line 42, in <module>
    from grow.common import ca_certs_locater
  File "/home/travis/.local/lib/python2.7/site-packages/grow/__init__.py", line 12, in <module>
    from .pods.pods import Pod
  File "/home/travis/.local/lib/python2.7/site-packages/grow/pods/pods.py", line 19, in <module>
    from grow.preprocessors import preprocessors
  File "/home/travis/.local/lib/python2.7/site-packages/grow/preprocessors/preprocessors.py", line 1, in <module>
    from . import blogger
  File "/home/travis/.local/lib/python2.7/site-packages/grow/preprocessors/blogger.py", line 6, in <module>
    from protorpc import messages
  File "/home/travis/.local/lib/python2.7/site-packages/protorpc/messages.py", line 1146, in <module>
    class Field(six.with_metaclass(_FieldMeta, object)):
TypeError: Error when calling the metaclass bases
    metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
```

Related to: https://github.com/grow/grow/issues/546

**Comment by pbakaus** [comment]
I love how the error is basically a tongue breaker: "metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases"

**Comment by pbakaus** [comment]
Fixed via grow/grow#546
